metricbeat/docs/modules/azure.asciidoc

```diff
@@ -18,6 +18,9 @@ Additional azure API calls will be executed in order to retrieve information reg
 
 The azure module mericsets are `monitor`,  `compute_vm` and `compute_vm_scaleset`
 
+IMPORTANT: Extra Azure charges on metric queries may be generated by this module.
+Please see <<<<azure-api-cost,additional notes about metrics and costs>> for more details.
```

Comment:
@kaiyan-sheng I think you have a typo here with  the extra `<<` chars

Reply:
ouch! I will fix it in a new PR! Thanks!!